bin/test_client.rb

```diff
@@ -99,6 +173,22 @@ def field_should_be_nil(fields, obj, method_name)
   end
 end
 
+def expected_fields_in_template_response
+  %w(id
+     type
+     created_at
+     created_by
+     body
+     version
+)
+end
+
+def expected_fields_in_template_preview
+  %w(id
+     body
+     version
+)
+end
```

Comment:
Also do we have a check to see that the `subject` exists for an `email` preview?